cmd/ui/src/ducks/auth/authSlice.ts

```diff
@@ -228,12 +228,12 @@ export const authExpiredSelector = createSelector(
 export const fullyAuthenticatedSelector = createSelector(
     (state: AppState) => state.auth,
     (authState) => {
-        if (authState.user === null || authState.sessionToken === null || authState.isInitialized === false) {
+        if (!authState.user || !authState.sessionToken || authState.isInitialized === false) {
```

Comment:
we also want to make sure `authState.user` is not undefined

Reply:
good call!